### ISSUE TITLE
[Feature]: Adjust the style of separators in the main menu and context menu

### ISSUE THREAD

**Issue by Inkiny** [mallowigi/material-theme-issues#373]
### Describe the feature

When I enable the Material Theme UI plugin, if I choose a built-in theme in my IDE, such as "Dark", the separators in the main menu and context menu appear as thin lines that match the border.
- Using the built-in theme "Dark" in the IDE, look like below images:
<image src="https://github.com/mallowigi/material-theme-issues/assets/140680029/252c5ac3-7a05-4188-bcc6-e6df0644d172" title="Dark - Main Menu" width="40%"/>
<image src="https://github.com/mallowigi/material-theme-issues/assets/140680029/37a5cf27-3e61-4cf7-9547-cfdd5b28aa75" title="Dark - Context Menu" width="40%"/>

However, if I select a theme from within this plugin, such as "Atom One Dark (Material)", the separators in the main menu and context menu become taller and look somewhat "ugly".
- Using the built-in theme "Atom One Dark (Material)" in this plugin, look like below images:
<image src="https://github.com/mallowigi/material-theme-issues/assets/140680029/944a12dc-5f3a-4a33-8d95-3464439a2876" title="Atom One Dark (Material) - Main Menu" width="40%"/>
<image src="https://github.com/mallowigi/material-theme-issues/assets/140680029/a6fc79ee-e5ea-45d9-a983-73ffa8bb6c7d" title="Atom One Dark (Material) - Context Menu" width="40%"/>

Is this a bug or a feature? I would appreciate it if you could add a feature to adjust these separator styles. Thank you!

### Material Theme Version

8.9.1

### IDE

IntelliJ IDEA

### System Info

```shell
IntelliJ IDEA 2023.2 (Ultimate Edition)
Build #IU-232.8660.185, built on July 26, 2023
Licensed to Bob
Subscription is active until June 31, 2024.
Runtime version: 17.0.7+7-b1000.6 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Registry:
    editor.focus.mode.color.light=5a5d63
    ide.editor.tab.selection.animation=true
    debugger.new.tool.window.layout=true
    ide.animate.toolwindows=true
    ide.new.editor.tabs.vertical.borders=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=5a5d63

Non-Bundled Plugins:
    com.intellij.spring.websocket (232.8660.142)
    com.chrisrm.idea.MaterialThemeUI (8.9.1)
    com.mallowigi (91.0.0)

Kotlin: 232-1.9.0-IJ8660.185
```

**Comment by mallowigi** [comment]
Yes they look taller, I never noticed... I'll check how that came to be and reduce them if I can
